### ISSUE TITLE
Rate and Review after dining

### ISSUE THREAD

**Issue by Huanjin-Zhang** [gcivil-nyu-org/nyc-accessible-restaurant-advisor#3]
**User Story**
- As a user, I want to evaluate restaurants by reviewing and rating various subjective aspects of the dining experience.
- As a user, I want to assess restaurants' accessibility by reviewing and rating the accessibility checklist.

**Acceptance criteria**
- If users are on the restaurant's page, they should be able to review and rate the subjective aspects of the restaurants.
- If users are people with disabilities, they should be able to review and rate the accessibility checklist.

**Definition of Done**
- Users can review and rate various subjective aspects of the restaurants.
- Users can review and rate the restaurants' accessibility checklist.
- Unit tests passed.
- Code reviewed.
- Acceptance criteria met.
